src/related-manager.ts

```diff
@@ -33,11 +33,14 @@ class RelatedManager {
     this.entries = [...this._entries, lastPlayedEntry];
   }
 
-  private async playByIndex(index: number) {
+  private playByIndex(index: number) {
     if (this._areEntriesExternal) {
       this.player.setMedia({sources: this.entries[index]});
+      this.player.play();
```

Comment:
use autoplay config like:
`kalturaPlayer.configure({playback: {autoplay: true}})`

Reply:
If the user has autoplay=false in config we're overriding that without notifying them, so if they just call loadMedia or setMedia manually from that player it will still autoplay. So that's ok ?

Reply:
got it. actually, this is how playlist [does it ](https://github.com/kaltura/kaltura-player-js/blob/master/src/common/playlist/playlist-manager.js#L274). but you are right.